app/models/schedule.rb

```diff
@@ -40,6 +36,20 @@ def sections
     sections
   end
 
+  def sheet_data
+    # There's two levels of caching going on here.
+    # 1. "@sheet_data ||=" which caches in memory for subsequent calls to sheet_data() within the same Schedule instance / web request
+    # 2. "Rails.cache" which caches between requests (up until expiry)
+    @sheet_data ||= Rails.cache.fetch(cache_key, expires_in: 5.minutes) do
```

Comment:
don't you think 5 mins is too short to cache? maybe move to 10?

Reply:
I learned:

>limit of 500 requests per 100 seconds per project, and 100 requests per 100 seconds per user. Limits for reads and writes are tracked separately. There is no daily usage limit.

So I think we should be fine, and it makes it faster to get last-minute changes out to attendees 

https://developers.google.com/sheets/api/limits